commit 6d64013764012598d8906dc4210d23881365c11f (refs/changes/08/708/2)
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2016-10-17T18:41:36+01:00 (3 years ago)
    
    MANTA-2802 Manta networking validation script mispells length
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/networking/validate.js b/networking/validate.js
index 68f2aa9..3adfbf9 100644
--- a/networking/validate.js
+++ b/networking/validate.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2016, Joyent, Inc.
  */
 
 /*
@@ -108,7 +108,7 @@ function main()
 	if (!('network' in spec['marlin']))
 		fatal('marlin section missing network name');
 
-	for (i = 0; i < spec['azs'].lenght; i++) {
+	for (i = 0; i < spec['azs'].length; i++) {
 		az = spec['azs'][i];
 		if (!(az in spec['admin']))
 			fatal('missing information for az ' + az +
